integrations/gradle-plugin/src/main/groovy/org/robolectric/gradleplugin/DownloadAndroidSdks.groovy

```diff
@@ -0,0 +1,101 @@
+package org.robolectric.gradleplugin
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.Project
+import org.gradle.api.tasks.TaskAction
+
+class DownloadAndroidSdks extends DefaultTask {
+
+  @TaskAction
+  void run() {
+    GradlePlugin.Config config = getProject().getExtensions().getByType(GradlePlugin.Config.class);
+
+    def defaultSdks = loadPropertiesResourceFile("org.robolectric.GradlePlugin.sdks.properties")
+    def enabledSdks = figureSdks(config.sdks, defaultSdks)
+    Properties sdkDeps = new Properties()
+    enabledSdks.keySet().forEach { Integer apiLevel ->
+      def sdkCoords = enabledSdks.get(apiLevel)
+      if (sdkCoords instanceof File) {
+        sdkCoords = project.files(sdkCoords)
+      }
+      def sdkCfg = project.configurations.create("sdk$apiLevel")
```

Comment:
Should this configuration be something that is better namespaced to robolectric, i.e. `robolectricSdk$apiLevel`?